templates/gerrit/notedb.config

```diff
@@ -0,0 +1,8 @@
+[noteDb "changes"]
+	autoMigrate = false
```

Comment:
not sure, why you dislike online migration so much ;)

Reply:
Actually I just used the configuration from the live server and added it to the recipe. Its not necessary to migrate anything here anymore, I am find with the switch `autoMigrate` being set to `on` or `off`, what do you prefer - apparently `on`?. I ran into issues when setting up a local test system when indexing changes failed as this now needs the `notedb.config` as changes have been migrated there.

Reply:
I see, then sure, leave it like it is.